conformance/reports/v0.8.0/nginxinc-nginx-kubernetes-gateway.yaml

```
@@ -0,0 +1,38 @@
+apiVersion: gateway.networking.k8s.io/v1alpha1
+date: "2023-08-31T16:40:14Z"
+gatewayAPIVersion: TODO
```

Unfortunately we still need to manually set this
```suggestion
gatewayAPIVersion: v0.8.0
```